rc/qml/MediaViewer/PhotoEditorPage.qml

```diff
@@ -22,11 +22,11 @@ Page {
     id: page
     property string photo
     signal done(bool photoWasModified)
-
+    
     title: i18n.tr("Edit Photo")
-
+    
     head.backAction: Action {
-        iconName: "back"
+        iconName: "save"
```

Comment:
Yes. Is quite confusing at first. For example, before, you edited for mistake, to go back you logically press the 'back' button expecting you actually go back, in reality it will go forward, because it save the photo and conclude the editing process. Now it's much clearer